# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: sparc-sparcv9 directory fix
# DP: Author: David S. Miller <davem@davemloft.net>
# DP: Upstream status: Jakub Jelinek installing similar patch to glibc CVS
# DP: Status Details:
# DP: Date: 2007-08-15

--- sysdeps/sparc/sparc32/sparcv9b/memcpy.S	2001-04-06 15:33:45.000000000 -0700
+++ /dev/null	2007-07-22 19:22:07.000000000 -0700
@@ -1,4 +0,0 @@
-#define ASI_PNF     0x82
-#define ASI_BLK_P   0xf0
-#define XCC icc
-#include <sparc64/sparcv9b/memcpy.S>
--- sysdeps/sparc/sparc32/sparcv9v/memcpy.S	2006-03-01 01:17:40.000000000 -0800
+++ /dev/null	2007-07-22 19:22:07.000000000 -0700
@@ -1,2 +0,0 @@
-#define XCC icc
-#include <sparc64/sparcv9v/memcpy.S>
--- sysdeps/sparc/sparc32/sparcv9v/memset.S	2006-03-01 01:17:40.000000000 -0800
+++ /dev/null	2007-07-22 19:22:07.000000000 -0700
@@ -1,2 +0,0 @@
-#define XCC icc
-#include <sparc64/sparcv9v/memset.S>
--- /dev/null	2007-07-22 19:22:07.000000000 -0700
+++ sysdeps/sparc/sparc32/sparcv9/sparcv9b/memcpy.S	2007-08-15
15:20:52.000000000 -0700
@@ -0,0 +1,4 @@
+#define ASI_PNF     0x82
+#define ASI_BLK_P   0xf0
+#define XCC icc
+#include <sparc64/sparcv9b/memcpy.S>
--- /dev/null	2007-07-22 19:22:07.000000000 -0700
+++ sysdeps/sparc/sparc32/sparcv9/sparcv9v/memcpy.S	2007-08-15
15:20:55.000000000 -0700
@@ -0,0 +1,2 @@
+#define XCC icc
+#include <sparc64/sparcv9v/memcpy.S>
--- /dev/null	2007-07-22 19:22:07.000000000 -0700
+++ sysdeps/sparc/sparc32/sparcv9/sparcv9v/memset.S	2007-08-15
15:20:55.000000000 -0700
@@ -0,0 +1,2 @@
+#define XCC icc
+#include <sparc64/sparcv9v/memset.S>
